azext_iot/sdk/iothub/pnp_runtime/version.py

```diff
@@ -9,5 +9,5 @@
 # regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "2020-06-30-preview"
+VERSION = "2020-09-30"
```

Comment:
This is a future dated API version. Is it already GA'd?

Reply:
Asking this to ensure if we have to make a release before then, we don't break users using the PnP runtime commands.

Reply:
I've been told this version is indeed GA'd - reaching out to service team for final validation and will update once I have it.

Reply:
Update - tested successfully with the node sdk pnp device sample for `twin show`, `twin update`, and `invoke-command`

Reply:
OK great. Could you please also add an entry to HISTORY.rst under a new `0.11.0` version.